zerver/openapi/zulip.yaml

```diff
@@ -448,6 +448,206 @@ paths:
                           ],
                         "upload_space_used": 571946,
                       }
+  /drafts:
+    get:
+      operationId: fetch_drafts
+      tags: ["drafts"]
+      description: |
+        Fetch all drafts for the current user.
+
+        `GET {{ api_url }}/v1/messages`
```

Comment:
Endpoint URL is wrong :)

Reply:
How embarrassing :man_facepalming:. I should have double checked this, looks like I got a little lazy. Thanks for the careful review here!